Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -127,6 +127,25 @@ div.bocList table.bocListTable td.bocListTitleCell > span
   padding: calc(var(--remotion-themed-spacing) / 2) var(--remotion-themed-spacing);
 }
 
+div.bocList table.bocListTable th.bocListTitleCell > a > img,
+div.bocList table.bocListTable th.bocListTitleCell > span > img
```

Comment:
```suggestion
div.bocList table.bocListTable th.bocListTitleCell > a > img,
div.bocList table.bocListTable td.bocListTitleCell > a > img,
div.bocList table.bocListTable th.bocListTitleCell > span > img,
div.bocList table.bocListTable td.bocListTitleCell > span > img
```

Reply:
Same here. This is for the column title icons/text so there is no reason to apply the style to `td`.